BambooTracker/gui/instrument_editor/visualized_instrument_macro_editor.cpp

```diff
@@ -406,7 +406,7 @@ void VisualizedInstrumentMacroEditor::drawLoop()
 					painter.fillRect(w, loopY_, 2, fontHeight_, palette_->instSeqLoopEdgeColor);
 					QString times = (loops_[j].times == 1) ? "" : QString::number(loops_[j].times);
 					painter.drawText(QRectF(w + 2, loopY_, colWidth - 2, fontHeight_),
-									 Qt::AlignLeft + Qt::AlignVCenter, tr("Loop %1").arg(times));
+									 Qt::AlignLeft | Qt::AlignVCenter, tr("Loop %1").arg(times));
```

Comment:
Qt flags should be combined via bitwise OR operations, adding them throws a compile-time exception with Qt6.